includes/blocks/class-sensei-course-navigation-block.php

```diff
@@ -0,0 +1,294 @@
+<?php
+/**
+ * File containing the Sensei_Course_Navigation_Block class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Sensei_Course_Navigation_Block
+ */
+class Sensei_Course_Navigation_Block {
+
+	/**
+	 * Settings placeholder.
+	 *
+	 * @var array
+	 */
+	private $settings;
+
+
+	/**
+	 * Sensei_Course_Navigation_Block constructor.
+	 */
+	public function __construct() {
+
+		$this->register_block();
+
+		$this->settings = [ 'collapsibleModules' => true ];
```

Comment:
Since we decided to remove this setting, I think we can remove this too, right?

Reply:
Removed in d17bdc8